packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -6,7 +6,7 @@
 
 import { ANALYTICS } from '@suite-actions/constants';
 import { Dispatch, GetState, AppState } from '@suite-types';
-import { getAnalyticsRandomId } from '@suite-utils/random';
+import { getAnalyticsRandomId } from '@trezor/utils';
```

Comment:
:thinking: I would suggest following: 

@trezor/utils/random
- move there only `getRandomId` function and renema it to something like `getWeakRandomId`.

@suite-utils/random
- keep getAnalyticsRandomId there as a pre-parametrized variant of `getWeakRandomId`. I would like to keep utils quite agnostic of other packages, so I suggest that it should not contain any reference to suite specifics (analytics here)

Reply:
It makes a lot of sense, thanks for catching it. I updated.